stan/math/prim/mat/prob/neg_binomial_2_log_glm_lpmf.hpp

```diff
@@ -22,6 +23,19 @@
 namespace stan {
 namespace math {
 
+namespace internal {
+template <typename T>
+inline T try_to_array_else_identity(T t) {
```

Comment:
Do you think we'll ever need this elsewhere? If so, we should move it to someplace where it's easy to get access to it.